src/components/header.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+
+import LoginButton from "./login-button";
```

Comment:
Might be nitpicky, but can we use absolute imports everywhere instead of relative imports? `~` is set as the the base url  (`src`).

```javascript
import LoginButton from "~/components/login-button";
```

This will prevent any relative import hell that we may face in the future.